examples/seesaw_ano_rotary_7segment_demo.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-FileCopyrightText: 2021 John Furcean
+# SPDX-License-Identifier: MIT
+
+"""I2C ANO rotary encoder with 7 segment display example."""
+
+import board
+from adafruit_ht16k33 import segments
+from adafruit_seesaw import seesaw, rotaryio, digitalio
+
+# For use with the STEMMA connector on QT Py RP2040
+# import busio
+# i2c = busio.I2C(board.SCL1, board.SDA1)
+# seesaw = seesaw.Seesaw(i2c, 0x49)
+
+i2c = board.I2C()  # uses board.SCL and board.SDA
+# i2c = board.STEMMA_I2C()  # For using the built-in STEMMA QT connector on a microcontroller
+seesaw = seesaw.Seesaw(i2c, addr=0x49)
+display = segments.Seg14x4(i2c, address=0x70)
+
+seesaw_product = (seesaw.get_version() >> 16) & 0xFFFF
+print("Found product {}".format(seesaw_product))
```

Comment:
Looks like there's a remaining use of `format()` for each example.

Reply:
You could even use `print("Found product", seesaw_product)` here too.

Reply:
whoops! missed those